README.md

```diff
@@ -12,70 +12,23 @@ PySTAC is a library for working with [SpatioTemporal Asset Catalog](https://stac
 
 ## Installation
 
-PySTAC requires Python >= 3.8. This project follows the recommendations of
-[NEP-29](https://numpy.org/neps/nep-0029-deprecation_policy.html) in deprecating support
-for Python versions. This means that users can expect support for Python 3.8 to be
-removed from the `main` branch after Apr 14, 2023 and therefore from the next release
-after that date.
```

Comment:
This feels appropriate to move to `installation.rst`, as it's good information to have but not needed at the top level README.